WordPress/src/main/java/org/wordpress/android/ui/quickstart/InitialTutorialPrompts.kt

```diff
@@ -0,0 +1,48 @@
+package org.wordpress.android.ui.quickstart
+
+import org.wordpress.android.R
+import org.wordpress.android.fluxc.store.QuickStartStore.QuickStartTask
+
+/**
+ * Static data about initial tutorial prompts you see when selecting one from Quick Start task list
+ **/
+enum class InitialTutorialPrompts constructor(
```

Comment:
Uh, I suck at naming, and renamed this class many times :) Initial here means that they are the first steps of quick start tasks. Tutorial probably can go :) Still not sure what is the good name for it. `MySite` fragment and `WPMainActivity` that hosts it contains the majority of Quick Start logic, so it felt right to consolidate related resources in some static way. At other screens, you can directly reference resources (they usually have only one Quick Start prompt), so having a way to pick the right one is not that important. I think we can consolidate them all here, add the static location of the prompt (ex. `PublicizeListFragment.class`) to help pick the right one (and rename it to `QuickStartPrompts`). What do you think? Any ideas are welcome :)

Reply:
> it felt right to consolidate related resources in some static way

I agree.  That makes sense to me too.

> What do you think?

I think having `QuickStart` as the prefix and anything like `Prompts`, `Data`, or `Source` is fine.  It seems like a good idea to have `QuickStart` in there if it's only going to used for Quick Start like the other classes to avoid confusion later.